src/WorkItemMigrator/WorkItemImport/WitClient/WitClientWrapper.cs

```diff
@@ -30,22 +31,39 @@ public WitClientWrapper(string collectionUri, string project, string personalAcc
             TeamProject = ProjectClient.GetProject(project).Result;
         }
 
-        public WorkItem CreateWorkItem(string wiType)
+        public WorkItem CreateWorkItem(string wiType, DateTime createdDate = default, string createdBy = "")
```

Comment:
use null instead of default

Reply:
fixed by https://github.com/solidify/jira-azuredevops-migrator/pull/648/commits/4f6ee65a7c91eaa61a952ca35f3b40df31fcb569